channel/code/app.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<app type="complex" access="777">
+  <version type="unsignedInt" length="4">1</version>
+  <os_version type="hexBinary" length="8">000500101000400A</os_version>
+  <title_id type="hexBinary" length="8">0005000014946100</title_id>
+  <title_version type="hexBinary" length="2">0000</title_version>
+  <sdk_version type="unsignedInt" length="4">20811</sdk_version>
+  <app_type type="hexBinary" length="4">80000000</app_type>
+  <group_id type="hexBinary" length="4">000004691</group_id>
+  <os_mask type="hexBinary" length="32">0000000000000000000000000000000000000000000000000000000000000000</os_mask>
+</app>
```

Comment:
Maybe all the channel stuff can be moved to the meta folder alonside the aroma/hbl meta?

Reply:
Yeah will do I dunno why I didn’t